packages/js/src/MatomoTracker.ts

```diff
@@ -109,6 +107,50 @@ class MatomoTracker {
     }
   }
 
+  // Tracks events based on data attributes
+  trackEvents() {
+    const matchString = '[data-matomo-event="click"]'
+    let firstTime = false
+    if (!this.mutationObserver) {
+      firstTime = true
+      this.mutationObserver = new MutationObserver((mutations) => {
+        for (const mutation of mutations) {
+          // Iterate over NodeList by indices (es15 does not allow using 'let node of mutation.addedNodes')
+          for (let ni = 0; ni < mutation.addedNodes.length; ni++) {
```

Comment:
I think we can rather use `Array.forEach` here: `mutation.addedNodes.forEach((addedNode) => ...)`, then we don't need to use `ni` and it's slightly more readable imho

Reply:
I think that should be possible if we run in through an `Array.from`

Reply:
`Array.forEach` did it. `Array.from` has a different purpose